js/api.ts

```diff
@@ -939,5 +929,5 @@ export class Client {
 }
 
 export function sleep(time) {
-    return new Promise(function(resolve) { return setTimeout(resolve, time); })
+  return new Promise(function(resolve) { return setTimeout(resolve, time); })
```

Comment:
Do we not want to use await? Such as:

```
export function sleep(ms) {
  await new Promise(function(resolve) {
    setTimeout(resolve, ms)
  })
}
```

Reply:
currently, await and async are not supported in otto engine.

Reply:
@masco-enovance moreover I think this function should be removed as not used anymore ?

Reply:
replaced with otto's sleep function